fastlane/Fastfile

```diff
@@ -326,21 +326,26 @@ lane :validate_docs do
 
   # Test if generating the docs is successful
   clone_docs do
+    # Test if generating the docs is successful
     generate_markdown_docs(target_path: ".")
```

Comment:
Could we just move this `generate_markdown_docs(target_path: ".")` into the `verify_docs` lane? And then just call the 'verify_docs` instead of doing the clone here?

Reply:
This would change what `verify_docs` does. As it is a public lane, I wasn't sure if someone/something is depending on this behavior and tried _not_ to change it. Because verifying something and generating some more into something is a bit different. Thoughts?